### PR TITLE
Add decimal time

### PR DESCRIPTION
**Pull request by mattkasun**
Addresses Issue #179 

display status/report in either default `8h 24min` or decimal `8.4h` or both `8h 24min  8.4h`

### CODE REVIEW

**Comment by dominikbraun** [comment]
Thanks for implementing this, @mattkasun! I'm going to review this PR soon.

**Review by dominikbraun — COMMENTED** [review]
This looks pretty good!

File: config/config.go [hunk 1]
```diff
@@ -5,12 +5,21 @@ import (
 	"github.com/spf13/viper"
 )
 
+type UseDecimalHours int
+
+const (
+	Off UseDecimalHours = iota
+	On
+	Both
+)
```
[the remark this answers is not in the document]
Comment (mattkasun) [reply config/config.go:14]: That is what i tried first but couldn't get it to work

**Comment by dominikbraun** [comment]
Nice! Feel free to disagree with me, but I personally prefer `both` over `2`. If there are no objections left, I'm going to merge this. 👍

**Review by dominikbraun — APPROVED** [review]

**Pull request merged**